localnet/kubernetes/values-relayminer-2.yaml

```diff
@@ -2,10 +2,8 @@ config:
   signing_key_name: supplier2
   suppliers:
     - service_id: anvil
-      type: http
       service_config:
-        url: http://anvil:8547/
-      proxy_names:
-        - http-proxy
-      hosts:
-        - tcp://relayminer2:8545
+        backend_url: http://anvil:8547/
+        publicly_exposed_endpoints:
+        - relayminer2:8545
```

Comment:
@red-0ne we don't need to have a port in `publicly_exposed_endpoints`, because the port is already specified in `listen_url`.